### DISCUSSION TITLE
WebSocket connection to 'wss://<URL>/_next/webpack-hmr' failed: websocket.js:80

### DISCUSSION THREAD

**Discussion by AlSajid** [vercel/next.js#53469] [Help]
### Summary

I am facing an issue with the production server of my Next.js application. The error shows in the browser console continuously like an infinite loop.

Edge shows:
`WebSocket connection to 'wss://<URL>/_next/webpack-hmr' failed: websocket.js:80`

Chrome shows:
`WebSocket connection to 'wss://<URL>/_next/webpack-hmr' failed: WebSocket is closed before the connection is established`

Firefox shows this message & freeze the webpage after few seconds.
`The connection to wss://<URL>/_next/webpack-hmr was interrupted while the page was loading.`

These errors only show when running in the hosted VPS server. This error does not show in localhost or Vercel hosting.
How can I fix it?

Fresh browser console without logging this error message & Firefox browser will not be frozen when user load this website.

**Comment by codeRuslan** [comment]
Hi @AlSajid, 

Did you manage to fix it?
